src/Goat/Update.elm

```diff
@@ -369,6 +369,25 @@ finishValidDrawing model drawingInfo =
             model
 
 
+foldDistance : Position -> ( Float, Position ) -> ( Float, Position )
+foldDistance position ( distance, previousPosition ) =
```

Comment:
`fold` is the higher order function, like `map` or `reduce`. So maybe this should be called sumDistances?